packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -53,6 +32,133 @@ describe('App: Settings', () => {
     cy.wait('@ReconfigureProject')
   })
 
+  describe('Project Settings', () => {
+    it('shows the projectId section when there is a projectId', () => {
+      cy.visitApp()
+      cy.findByText('Settings').click()
+      cy.findByText('Project Settings').click()
+      cy.findByText('Project ID').should('be.visible')
+    })
+
+    it('shows the Record Keys section', () => {
+      cy.loginUser()
+
+      cy.visitApp()
+      cy.findByText('Settings').click()
+      cy.findByText('Project Settings').click()
+      cy.findByText('Record Key').should('be.visible')
+    })
+
+    it('obfuscates each record key and has a button to reveal the key', () => {
+      cy.loginUser()
+
+      cy.visitApp()
+      cy.findByText('Settings').click()
+      cy.findByText('Project Settings').click()
+      cy.get('[data-cy="record-key"]').should('contain', '***')
+      cy.get('[aria-label="Record Key Visibility Toggle"]').click()
+      cy.get('[data-cy="record-key"]').should('contain', '2aaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa')
+    })
+
+    // TODO: Check that 'Default values' are indicated when the specPattern is not edited
```

Comment:
We're trying to cut down on TODOs that aren't explicitly linked to an issue. If this req is blocked from testing, can we log a followup and remove the TODO?

Reply:
Looking at the FIgma, we no longer indicate if the spec pattern is the default
https://www.figma.com/proto/PAOozuCxtfetqP8fUVvg0y/Cypress-App?node-id=1688%3A14486&scaling=scale-down&page-id=0%3A1&starting-point-node-id=1688%3A14486&show-proto-sidebar=1

Unless I'm missing something, we can just remove this comment and edit the test name.